frontend/javascripts/admin/project/project_list_view.js

```diff
@@ -288,6 +288,18 @@ class ProjectListView extends React.PureComponent<PropsWithRouter, State> {
                 width={200}
                 sorter={Utils.compareBy(typeHint, project => project.numberOfOpenAssignments)}
               />
+              <Column
+                title="Time [h]"
```

Comment:
Do you think "Time" is self-explanatory? Otherwise, we could add a tooltip saying "Total time spent annotating for this project".